connector/external/external.go

```diff
@@ -0,0 +1,59 @@
+// Package external implements a connector which relies on external
+// authentication (e.g. mod_auth in Apache2) and returns an identity with the
+// HTTP header X-Remote-User as verified email.
+package external
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/coreos/dex/connector"
+	"github.com/sirupsen/logrus"
+)
+
+// Config holds the configuration parameters for a connector which returns an
+// identity with the HTTP header X-Remote-User as verified email.
+type Config struct {
+	PathSuffix string `json:"pathSuffix"`
```

Comment:
I mean, this should be passed by the server since it know the URL this connector will operate at.

Reply:
Maybe let connectors know what their ID is?

```
func (c *Config) Open(id string, logger logrus.FieldLogger) (connector.Connector, error)
```

Reply:
Done.